osf/models/preprint.py

```diff
@@ -425,33 +423,6 @@ def can_view_files(self, auth=None):
         else:
             return self.can_view(auth=auth)
 
-    # Overrides ContributorMixin entirely
-    # TODO: When nodes user guardian as well, move this to ContributorMixin
-    def has_permission(self, user, permission):
```

Comment:
Many contributor-related methods on the preprint model moved to ContributorMixin, now that nodes are also using guardian.